.github/workflows/periodic.yml

```
@@ -26,3 +26,29 @@ jobs:
           title: Broken link detected by CI
           content-filepath: .github/ISSUE_TEMPLATE/broken-link.md
           labels: automated, broken link
+
+  test-bundle-periodic:
+    name: Operator Bundle Test
+    if: github.repository_owner == 'submariner-io'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+      - name: Fetch all git tags
```

```suggestion

      - name: Fetch all git tags
```

Maybe blanks between these sections?